metaflow/cmd/main_cli.py

```diff
@@ -70,6 +71,7 @@ def status():
 ]
 
 process_cmds(globals())
+diff.register_commands(main)
```

Comment:
Needs to be added like above with CMDS_DESC. Also, could we make one diff command and the other things sub commands (or maybe make it a subcommand of develop or develop diff although that may be too long).

Reply:
I had it as a subcommand but `metaflow diff diff` looked weird

Reply:
what does adding it to `CMDS_DESC` do? it seems to be working without doing that